srcpkgs/SDL/template

```diff
@@ -1,24 +1,24 @@
 # Template file for 'SDL'
 pkgname=SDL
 version=1.2.15
-revision=11
+revision=13
 patch_args="-Np1"
 build_style=gnu-configure
-configure_args="--enable-alsa --disable-esd --disable-rpath
+configure_args="--enable-alsa --enable-sndio --disable-esd --disable-rpath
  --enable-clock_gettime --disable-nas --disable-arts --disable-x11-shared
  --disable-alsa-shared --disable-pulseaudio-shared --disable-video-dga
  --disable-osmesa-shared $(vopt_enable aalib video-aalib)
  $(vopt_enable pulseaudio) $(vopt_enable opengl video-opengl)
  $(vopt_if x11 '-enable-video-x11-xrandr --enable-video-x11-vm --enable-video-x11-xv')"
 short_desc="Simple Directmedia Layer"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.libsdl.org/"
 license="LGPL-2.1"
```

Comment:
please fix license while you're at it

Reply:
I know this pr is closed, but what maxie means is fix it so it uses the proper spdx identifier. https://spdx.org/licenses/
It needs to be updated to either LGPL-2.1-only or LGPL-2.1-or-later

Reply:
Thank you. It's done!